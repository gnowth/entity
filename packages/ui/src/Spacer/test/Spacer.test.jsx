import renderer from 'react-test-renderer';
import React from 'react';

import UISpacer from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <UISpacer />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
