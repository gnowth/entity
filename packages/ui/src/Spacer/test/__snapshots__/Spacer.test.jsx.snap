// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

<div
  className="c0"
/>
`;
